assets/js/components/event/StreamArea.tsx

```diff
@@ -41,16 +41,15 @@ const StreamArea = ({
   chatMessages,
   isChatLoaded,
   isBannedFromChat,
+  attachVideo,
+  setSrc,
+  enablePictureInPicture,
 }: StreamAreaProps) => {
   const [amIPresenter, setAmIPresenter] = useState<boolean>(false);
   const [presenterName, setPresenterName] = useState<string>("");
-  const { attachVideo, setSrc } = useHls(true, config);
-  const { device, orientation } = useContext(ScreenTypeContext);
+  const screenType = useContext(ScreenTypeContext);
   const { setStreamStart } = useStartStream();
   const [card, setCard] = useState<CardStatus>("hidden");
-  const showMobileBottomBar = device === "mobile" || orientation === "portrait";
-
-  useAutoHideMobileBottomBar(setCard);
```

Comment:
Did you remove / revert this code accidentally or on purpose?

Reply:
By accident. I'll include that back in

Reply:
I've mended it.